front/lib/models/assistant/conversation.ts

```diff
@@ -492,7 +506,7 @@ Message.belongsTo(UserMessage, {
 });
 
 AgentMessage.hasOne(Message, {
-  as: "agentMessage",
+  as: "message",
```

Comment:
You want to also fix ContentFragmentModel below

Reply:
Done